src/lib/pipeline.ml

```diff
@@ -74,6 +74,7 @@ module Parameters = struct
     with_mutect2: bool [@default false];
     with_varscan: bool [@default false];
     with_somaticsniper: bool [@default false];
+    with_optitype: bool [@default false];
```

Comment:
>Not sure OTOMH how to resolve the confusion between seq2hla's results being fed through the rest of the pipeline

We could have some logic to detect if multiple HLA typers are run, compare the results, and highlight when we have a mismatch?

Of course we'll need logic on what to do upon mismatch--we could have a ranking of HLA typers by those we trust most I guess. I actually think OptiType is more accurate than seq2HLA from what I recall...